src/app/modules/browse/browse-controller.js

```diff
@@ -4,6 +4,7 @@
 	.config(['$stateProvider', function config($stateProvider) {
 
 		$stateProvider.state('browse', {
+            parent: 'requireLoginResolution',
```

Comment:
Something looks wrong with indent here...are you being consistent about indenting with spaces or tabs?